lib/express/myinfo/controllers.js

```diff
@@ -132,15 +132,11 @@ module.exports =
         type: 'application/x-www-form-urlencoded',
       }),
       (req, res) => {
-        const [tokenTemplate, redirectURI] =
-          consent.authorizations[req.body.code]
+        const [tokenTemplate] = consent.authorizations[req.body.code]
         const [, authHeader] = (req.get('Authorization') || '').split(' ')
 
         const { signature, baseString } = MYINFO_SECRET
-          ? myInfoSignature(authHeader, req, {
-              clientSecret: MYINFO_SECRET,
```

Comment:
doesn't `client_secret` still need to go in?

Reply:
oh right it's part of query params already. ignore me haha

Reply:
it might be, but I realised that for a realistic mock, we should compare the signature against one generated from a base string containing what mockpass knows as the client secret. I'm patching this now.